packages/plugin-ua-parser-browser/src/ua-parser-plugin.ts

```diff
@@ -0,0 +1,50 @@
+import { BrowserConfig, EnrichmentPlugin, Event } from '@amplitude/analytics-types';
+import { CreateUaParserPlugin, Options } from './typings/ua-parser-plugin';
+import UAParser from '@amplitude/ua-parser-js';
+
+export const uaParserPlugin: CreateUaParserPlugin = function (
+  options: Options = {
+    osName: true,
+    osVersion: true,
+    deviceManufacturer: true,
+    deviceModel: true,
+  },
```

Comment:
```suggestion
  options: Options = {},
```

the default values for each key should be taken care of by the next line